presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java

```diff
@@ -58,13 +58,13 @@ public void configure(Binder binder)
         binder.bind(JdbcConnector.class).in(Scopes.SINGLETON);
         configBinder(binder).bindConfig(JdbcMetadataConfig.class);
 
-        newExporter(binder).export(Key.get(JdbcClient.class, InternalBaseJdbc.class))
+        newExporter(binder).export(Key.get(JdbcClient.class, StatsCollecting.class))
```

Comment:
here we do export, so it's reasonable that we want the `JdbcClient` layer which collects stats (`StatsCollecting`) ...